core/secret_scrub.go

```diff
@@ -0,0 +1,10 @@
+package core
+
+// ToScrubInfo stores the info to access secrets and scrub them from outputs.
```

Comment:
nit: Comment doesn't match struct name

Reply:
Thanks for catching this one! Will fix it!

Reply:
done